apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -168,11 +169,12 @@ class SectionProgress extends Component {
               onChange={this.onChangeScript}
             />
           </div>
-          <div style={styles.toggle}>
-            <div style={{...h3Style, ...styles.heading}}>{i18n.viewBy()}</div>
-            <SectionProgressToggle />
-          </div>
-
+          {levelDataInitialized && (
```

Comment:
why does this need to be wrapped like this now?

Reply:
I did it that way because if we don't have `scriptData` yet we don't know whether it's a CSF script or not so we don't know whether to show standards or not so better to just what until we have all that info.